notebooks/network_mapping/custream_n_graph.ipynb

```diff
@@ -0,0 +1,386 @@
+{
```

Comment:
<p>Can you make an alternative output that is just the graph embedding? For some workflows we're not interested in outputing the pagerank, but instead the graph made from the edgelist</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/405/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Yes, what we really want to do is generate a graph representation and keep that updated per time slice. It would be good to also keep the analytic results around as well, but that is secondary.</p><p> </p>